pages/1.10/release-notes/1.10.10/index.md

```diff
@@ -0,0 +1,190 @@
+  - Ability to deploy to CNI-Based virtual networks.
+
+You can try out the new features and updated data services. Provide feedback through our support channel: <a href="https://support.mesosphere.com/">support.mesosphere.com</a>.
+
+## New Features and Capabilities
+
+### Networking
+- Configurable Spartan upstreams for domains (dnames).
+  You can now configure Spartan to delegate a particular domain (for example, `\*.foo.company.com`) to a particular upstream.
+
+- Increased CNI network support.
+  DC/OS now supports any type of CNI network. [View the documentation](/1.10/networking/virtual-networks/cni-plugins/).
+
+- Edge-LB load balancer. [enterprise type="inline" size="small" /]
+
+  Edge-LB load balances Mesos tasks. Not supported in strict security mode. [View the documentation](/services/edge-lb/0.1/).
+
+[enterprise type="block" size="large"]
+### Security
+[/ enterprise]
+
+- Custom CA certificate support.
+  Installation time [configuration options](/1.10/security/ent/tls-ssl/ca-custom/) have been added that allow you to configure DC/OS Enterprise to use a custom CA certificate and corresponding private key, which DC/OS then uses for issuing all component certificates. The custom CA certificate can be an intermediate CA certificate so that that all certificates used within the DC/OS cluster derive from your organization’s X.509 certification hierarchy.
+
+- Enhanced secrets management with file-based secrets.
+  You can now make a secret available to your service in the sandbox of the task. [View the documentation](/1.10/security/ent/secrets/use-secrets/).
+
+- Vastly improved IAM scalability and performance characteristics.
+  The new system removes hard limits on the number of users, groups, and permissions that can be stored, and shows stable read and write performance as the dataset grows.
+
+- Docker `pullConfig` parameter.
+  Use this parameter in your service definition to authenticate to a private Docker registry. [View the documentation](/1.10/deploying-services/private-docker-registry/#referencing-private-docker-registry-credentials-in-the-secrets-store-enterprise).
+
+ - Enterprise CLI permissions management commands.
+   It is now possible to manage permissions to protect resources using the [DC/OS Enterprise CLI](/1.10/security/ent/perms-management/).
+
+### Kubernetes on DC/OS
+
+- Kubernetes on DC/OS is beta with DC/OS 1.10.0. Install from the DC/OS Service Catalog or use the [quickstart](https://github.com/mesosphere/dcos-kubernetes-quickstart).
+
+### Updated DC/OS Data Services
+
+- Rolling Configuration Update and Upgrades support via the CLI. [enterprise type="inline" size="small" /]
+- Ability to deploy Data Services into Folders to enable multi team deployments. [enterprise type="inline" size="small" /]
+- Ability to deploy to CNI-Based Virtual Networks.
+
+The following updated data services packages are compatible with DC/OS 1.10.
```

Comment:
Could you check and confirm with the SDK team if it is necessary to include the latest link to the release notes for each data service package that are compatible with DC/OS 1.10 OR if the following links are accurate to support the same?